### DISCUSSION TITLE
Problems working with Appcheck

### DISCUSSION THREAD

**Discussion by nelrafa103** [vuejs/vuefire#1371] [Questions and Help] [answered]
Is there any way to use appcheck properly ? Also there is not documentation on the page about it

**Comment by posva** [comment]
It works well in SPA pages. Until documentation is released you can check the playground.

- https://github.com/vuejs/vuefire/blob/main/playground/src/main.ts#L47
- https://github.com/vuejs/vuefire/blob/main/playground/src/pages/app-check.vue

**Reply by posva** [reply, marked as answer]
It's documented now! https://vuefire.vuejs.org/guide/app-check.html
